deploy-board/deploy_board/static/js/components/clusterconfigcomponents.js

```diff
@@ -585,12 +585,19 @@ Vue.component('backup-hosttypes', {
     <div class="col-xs-2"></div>\
     <div class="col-xs-8">\
         <span class="col-xs-8" style="padding:0;">Backup Host Types In Order: {{backuphosttypes}}</span>\
+        <form-warning :alert-text="backupTypesWarning"></form-warning>\
     </div>\
     </div>',
     props: ['backuphosttypes'],
     computed: {
         isVisible: function () {
             return this.backuphosttypes !== 'None'
+        },
+        backupTypesWarning: function () {
+            const part1 = "You should only enable this feature with capacity-eng approval. ";
+            const part2 = "Your enable/disable operation on this feature triggers a notification to capacity-eng.";
+            text = part1 + part2;
```

Comment:
Shouldn't this be `const text`?
```suggestion
            const text = part1 + part2;
```